src/__tests__/featureflags.ts

```diff
@@ -32,6 +32,7 @@ describe('featureflags', () => {
             get_property: (key) => instance.persistence.props[key],
             capture: () => {},
             decideEndpointWasHit: false,
+            receivedFlagValues: false,
```

Comment:
In this PR we are essentially find+replacing the existing flag `decideEndpointWasHit` for this new, more accurately named `receivedFlagValues` flag. The prior will now mean that the endpoint truly was hit with no errors, and the latter will mean that there are flag values available for the SDK to return. 
`decideEndpointWasHit` is now really only used for passing back the `$used_bootstrap_value` flag in $feature_flag_called events.